spec/deaf_grandma_spec.rb

```diff
@@ -10,9 +10,11 @@
   
   it "says 'NOT SINCE 1964!' when we yell" do
     #implement your test here
+    expect(script.speak("HI GRANDMA")).to eq "NOT SINCE 1964!"
   end
 
   it "EXTRA CREDIT: How would you test yelling BYE?" do
     #implement your test here
+    2.times{ expect(script.speak("BYE")).to eq nil }
```

Comment:
Yelling BYE twice should result in granny responding with "NOT SINCE 1964!", and something different should happen when you yell it a third time.